ktor-client/ktor-client-core/build.gradle.kts

```diff
@@ -12,6 +12,7 @@ kotlin.sourceSets {
             api(project(":ktor-http"))
             api(project(":ktor-http:ktor-http-cio"))
             api(project(":ktor-shared:ktor-events"))
+            api(project(":ktor-shared:ktor-serialization"))
```

Comment:
why is it needed here?

Reply:
`DefaultClientWebSocketSession` located in `client-core`, so new methods located there too and `ktor-shared:ktor-serialization` needed. Should I locate this methods in another module?

Reply:
no, it's good like this, thanks for explaining